<launch>

<!-- Rocam Control Node -->
	<node pkg="rocam" type="rocam_control" name="control_ROCAM_motion"/>

<!-- joy node -->
	<node respawn="true" pkg="rocam" type="joy_rocam" name="joy_rocam" >
		<param name="dev" type="string" value="/dev/input/js0" />
		<param name="deadzone" value="0.05" />
	</node>

	<!-- Axes -->
		<param name="axis_linear" value="1" type="int"/>
		<param name="axis_angular" value="0" type="int"/>
		<param name="scale_linear" value="2" type="double"/>
		<param name="scale_angular" value="2" type="double"/>

</launch>
